app/javascript/controllers/modal_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="modal"
+export default class extends Controller {
+  static targets = [ "modal", "overlay" ]
+  connect() {
+    console.log("hello");
+  }
+
+  open() {
+    this.modalTarget.classList.remove('d-none');
+    this.overlayTarget.classList.add('overlay');
+  }
```

Comment:
I also found this online: `this.modalTarget.focus(); `

Apparently, you can shift the focus to accessibility: move the focus to the modal. But I would also need to read more on this 🤓

Reply:
Did you look at this?

Reply:
no sorry, this was hidden comments I didnt see I'll look at it now haha